README.md

```diff
@@ -63,6 +63,7 @@ No.
 | he_mount_options | '' | NFS mount options
 | he_storage_domain_path | null | shared folder path on NFS server |
 | he_nfs_version | auto | NFS version.  available options: *auto*, *v4*, *v3*, *v4_0*, *v4_1*, *v4_2*
+| he_storage_if | null | If the separate network interface of storage was pre-configured - will be check operstate|
```

Comment:
not really clear what the parameter defines, let's describe it more plainly, something like: `interface name that is connected to the storage network, assumed to be pre-configured. `